server/plugin/template.go

```diff
@@ -440,6 +442,16 @@ It now has **{{.GetRepo.GetStargazersCount}}** stars.`))
 {{- if eq .GetAction "created" }} created a release {{template "release" .GetRelease}}
 {{- else if eq .GetAction "deleted" }} deleted a release {{template "release" .GetRelease}}
 {{- end -}}`))
+
+	template.Must(masterTemplate.New("newDiscussion").Funcs(funcMap).Parse(
+		`Discussion [#{{.GetNumber}} {{.GetTitle}}]({{.GetHTMLURL}})`,
```

Comment:
Can we make this little more descriptive, like:

`@user stater a new discussion #number title on [org/repo]`

Reply:
updated both templates in commit 3a637da